Lib/test/test_context.py

```diff
@@ -361,6 +358,10 @@ def sub(num):
             tp.shutdown()
         self.assertEqual(results, list(range(10)))
 
+    def test_contextvar_getitem(self):
+        clss = contextvars.ContextVar
```

Comment:
just for my own curiosity, why did you prefer aliasing this? It looks like it doesn't exceed the char limitation.

Reply:
in my opinion it has better readability and less complexity :man_shrugging: